Makefile

```diff
@@ -46,6 +46,43 @@ unit_coverage: unit ## Runs unit tests and generates a html coverage report
 integration_coverage: integration ## Runs integration tests and generates a html coverage report
 	go tool cover -html="$(PROJECT_PATH)/_output/integration.cov"
 
+
+### end-to-end
+
+.PHONY: e2e.environment
+e2e.environment: e2e.cluster-up e2e.install-istio e2e.sample-app # Setup local k3s cluster with Istio, 3scale adapter, and httpbin sample app
+
+.PHONY: e2e.get-istio
+e2e.get-istio: ## Fetch istio release templates - Specify version as ISTIO_VERSION
+	@if [ -z ${ISTIO_VERSION} ]; then echo ISTIO_VERSION is unset && exit 1; fi
```

Comment:
Both here and in the scripts it would be beneficial to be able to extract the istio version from the library dependencies we have declared in dep/go mod - perhaps we can write a quick script to extract it. This way we don't depend on a manual spec or the latest release, one being error prone and the latter almost sure to not be the right version.